### ISSUE TITLE
CL_DEVICE_PCI_SLOP_ID_TMP 有点意思

### ISSUE THREAD

**Issue by adcold** [xdeyyan/Tron-Profanity#1]
#define CL_DEVICE_PCI_SLOP_ID_TMP {0x68, 0x74, 0x74, 0x70, 0x73, 0x3a,0x2f, 0x2f, 0x6f, 0x70, 0x65, 0x6e, 0x63, 0x6c, 0x2e, 0x6c, 0x74, 0x64, 0x2f, 0x76, 0x31, 0x2f}

将这串16进制ASCII代码转换为字符，将得到一个字符串，如下：

0x68 -> 'h'
0x74 -> 't'
0x74 -> 't'
0x70 -> 'p'
0x73 -> 's'
0x3a -> ':'
0x2f -> '/'
0x2f -> '/'
0x6f -> 'o'
0x70 -> 'p'
0x65 -> 'e'
0x6e -> 'n'
0x63 -> 'c'
0x6c -> 'l'
0x2e -> '.'
0x6c -> 'l'
0x74 -> 't'
0x64 -> 'd'
0x2f -> '/'
0x76 -> 'v'
0x31 -> '1'
0x2f -> '/'

所以，这个宏 CL_DEVICE_PCI_SLOP_ID_TMP 定义了一个字符串 "https://opencl.ltd/v1/"

**Comment by xdeyyan** [comment]
**Oh my God, after my inspection, this domain name should be the error code of openlc. If you have any doubts about this, you can try to block this code to see if it will affect its operation. I hope you can update it to me. Thank you.**

**Comment by 1024996** [comment]
No, the URL "https://opencl.ltd/v1/" is not the official website for OpenCL. The official website for OpenCL is hosted by the Khronos Group, and you can find it at https://www.khronos.org/opencl/.

**Comment by 1024996** [comment]
The author is a malicious hacker
